patroni/postgresql/connection.py

```diff
@@ -147,7 +147,7 @@ def get(self, name: str, kwargs_override: Optional[Dict[str, Any]] = None) -> Na
     def close(self) -> None:
         """Close all named connections from Patroni to PostgreSQL registered in the pool."""
         with self._lock:
-            if any(conn.close(True) for conn in self._connections.values()):
+            if all(conn.close(False) for conn in self._connections.values()):
```

Comment:
We don't want to have a separate log line for every closed connection,  but a single line indicating that one or more connections have been closed.
It could only be achieved by first executing close() method on all connections, accumulating results to a list, and checking if any of the elements in the list contain True.